Sources/SwiftProtobuf/NameMap.swift

```diff
@@ -43,21 +41,24 @@ private func toJsonFieldName(_ s: String) -> String {
 /// string data.  Track the buffers and release all of those buffers
 /// in case we ever get deallocated.
 fileprivate class InternPool {
-  private var interned = Set<UnsafePointer<UInt8>>()
+  private var interned = [UnsafeBufferPointer<UInt8>]()
 
   func intern(utf8: String.UTF8View) -> UnsafeBufferPointer<UInt8> {
     let count = utf8.count
-    let raw = malloc(count)!
+    let raw = UnsafeMutableRawPointer.allocate(bytes: count, alignedTo: 1)
```

Comment:
`UnsafeMutablePointer.initialize()` won't work due to:

> warning: 'initialize(from:)' is deprecated: it will be removed in Swift 4.0.  Please use 'UnsafeMutableBufferPointer.initialize(from:)' instead

but I did manage to simplify it a bit.